session_items.py

```diff
@@ -1,75 +1,34 @@
 from flask import session
-
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
-
+import requests
```

Comment:
As this file no longer used the Flask session cookie I'd advocate changing its name to something like `trello_api.py`

Reply:
Renamed to "board_api" to abstract away the fact we're using Trello as our underlying store.